### ISSUE TITLE
restore window layout in neovim terminal mode

### ISSUE THREAD

**Issue by ianks** [janko-m/vim-test#146]
There is a similar issue with vim-test to the one described here: https://github.com/junegunn/fzf/issues/520

Ideally, vim-test would restore the window layout to the way it was before the test results buffer is closed. I am opening this issue to begin tracking the problem/finding the solution.

**Comment by janko-m** [comment]
@ianks Thanks for linking the FZF issue, that's really helpful! From skimming it it looks like we could use the same solution for test.vim. I will probably have time next week to read the issue more in depth, and probably add the same solution to test.vim.

**Comment by ianks** [comment]
@janko-m Have you looked at this at all?

**Comment by janko-m** [comment]
@ianks Nope, not yet. It seems that we should be able to easily copy the logic from fzf.vim, I just don't have the time lately.

**Comment by ianks** [comment]
Ok let me see if I can do something.

**Comment by janko-m** [comment]
@codeinabox This is an issue that most likely can be solved, and we have a good path forward (copying logic from fzf.vim), it's just that nobody had the time to dig into that yet. So I would prefer to keep it open, unless we realize that fzf.vim hasn't solved it either.

**Comment by ianks** [comment]
I messed around with it a bit, but my vimscript skills are not up to par to make it work robustly. I think the solution is pretty straightforward though.

**Comment by ianks** [comment]
Can we put bounties on issues? I would happily contribute to make this happen.
